packages/three-vrm/src/nodes/index.ts

```diff
@@ -0,0 +1 @@
+export * from '@pixiv/three-vrm-materials-mtoon/nodes';
```

Comment:
Should we and some three version check and log info here or inside `@pixiv/three-vrm-materials-mtoon/nodes`? IMO it's not necessary

Reply:
we already have a warning inside `MToonNodeMaterial` so I think it's not needed

Reply:
I think MToonNodeMaterial'constructor will never be executed if `three/addons/nodes/Nodes` not exist?

Reply:
Ahh, yes it's possible to cause an error before `MToonNodeMaterial` instantiates.

Does this help?

![image](https://github.com/pixiv/three-vrm/assets/7824814/dea1f584-4f4d-4adc-8056-c737688bafa4)